### DISCUSSION TITLE
No Backdrop Image

### DISCUSSION THREAD

**Discussion by vladimirgluten** [SFX-WoW/Masque_Caith#31] [Issues] [answered]
I see Masque got updated today and when I logged in the buttons in Dominos do not have the backdrop.  Enabling the backdrop just results in some buttons with a dark blank background and others with a more grey semi-transparent one.  Tried disabling both Dominos and Masque and re-enabling, removing Masque and reinstalling and restarting the game.  

![masque](https://user-images.githubusercontent.com/27385251/64576850-082c4f00-d340-11e9-8138-f31349f502fa.PNG)

**Comment by StormFX** [comment]
_Caith_ doesn't have a textured background anymore.

**Comment by StormFX** [comment]
I did notice that some backgrounds are more transparent and I've found the problem.

**Comment by StormFX** [comment, marked as answer]
Marking this as a duplicate of #3.
